### PR TITLE
Link credentials in profile

### PR DESCRIPTION
**Pull request by huwshimi**
Link the credentials in the model page to the credentials page.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- load the new profile
- expand a model row
- click on the credential name
- it should take you to the credentials section of the profile (in the future this should probably do something like highlight the correct row)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8425/

File: jujugui/static/gui/src/app/components/profile/model-list/model-list.js [hunk 1]
```diff
@@ -126,6 +126,16 @@ class ProfileModelList extends React.Component {
     this._showConfirmation(model);
   }
 
+  /**
+    Navigate to a credential.
+    @param credential {String} The credential name.
+  */
+  _handleCredentialClick(credential) {
+    this.props.changeState({
+      hash: `credentials/${credential}`
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/profile/model-list/model-list.js]: nice idea

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Comment by cassiocassio** [comment]
We don’t have an expanded style for creds because there’s not much useful
extra stuff to show.

It doesn’t feel right to open the creds in update/edit mode as that’s a
rare action.

What’s the right high lit style? @spencer?

**Pull request merged**